tests/store/tracking/test_file_store.py

```diff
@@ -402,10 +409,14 @@ def test_create_experiment(self):
             exp1.artifact_location,
             path_to_local_file_uri(posixpath.join(self.test_root, created_id)),
         )
+        assert exp1.creation_time > test_case_start_time
```

Comment:
```suggestion
        assert exp1.creation_time >= test_case_start_time
```

Reply:
In delete/restore/rename operation, we need sleep to ensure `last_update_time` has been updated, but in create operation, we don't.